### PR TITLE
fix docs for saving variable in context

### PR DESCRIPTION
**Pull request by sih4sing5hong5**
for `response = context.response # from previous step` in line 87

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -73,7 +73,7 @@ testing client.
     # Using Django's testing client
     @when(u'I visit "{url}"')
     def visit(context, url):
-        response = context.test.client.get(url)
+        context.response = context.test.client.get(url) # save response in context for next step
```
Comment (bittner) [inline docs/usage.rst]: Please put the comment (if really necessary) in a line above the code: Let's do no in-line comments if that can be avoided.

And when you're at it, can you please also modify lines 87 and 88 as follows? (`@then` and comment)
``` python
     @then(u'I should see "{text}"')
     def visit(context, text):
         # compare with response from ``when`` step
         context.test.assertContains(context.response, text)
```
Thank you for your help and understanding!

**Comment by bittner** [comment]
@sih4sing5hong5 There are now merge conflicts due to other processed PRs. You probably best reapply your changes (manually) to a fresh fork of this repo. Do you want to go ahead?

**Comment by sih4sing5hong5** [comment]
ok, I will finish it.

**Comment by bittner** [comment]
Thank you, @sih4sing5hong5!

**Pull request merged**